src/Microsoft.Azure.WebJobs.Host/Scale/ProcessMonitor.cs

```diff
@@ -91,13 +89,20 @@ public virtual ProcessStats GetStats()
 
         private void OnTimer(object state)
         {
-            if (_disposed || _process.HasExited)
```

Comment:
This was the point of the bug I discovered. When running in the Functions Host, if I for example killed the OOP Python worker, sometimes HasExited would actually throw because the process had been fully cleaned up. So I'm adding error handling for that case (see ProcessWrapper.HasExited).